.github/workflows/test-application.yaml

```
@@ -0,0 +1,77 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+
+jobs:
+    test:
+        name: "PHP ${{ matrix.php-version }} (dependency ${{ matrix.dependency-versions }})"
+        runs-on: ubuntu-latest
+
+        env:
+            DATABASE_URL: mysql://root:root@127.0.0.1:3306/sulu_test?serverVersion=5.7
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.2'
+                      php-extensions: 'ctype, iconv, mysql, gd'
+                      dependency-versions: 'locked'
```

i guess `lowest` does not work right? 🙂 
`locked` is basically the same as `highest` if there is not `composer.lock`, so wi would just use `highest`

```suggestion
                      dependency-versions: 'highest'
```